rmgpy/molecule/molecule.py

```diff
@@ -627,16 +628,50 @@ def sorting_key(self):
                 self.atom1.number if self.atom1 is not None else 0,
                 self.atom2.number if self.atom2 is not None else 0)
 
-    def get_bde(self):
+    def get_bde(self, metal='Pt111'):
         """
         estimate the bond dissociation energy in J/mol of the bond based on the order of the bond
         and the atoms involved in the bond
+
+        If the bond involves a surface site, the atomic binding energies from the `metal` (str) 
+        will be use to estimate the bond dissociation energy.
         """
         try:
             return bdes[(self.atom1.element.symbol, self.atom2.element.symbol, self.order)]
         except KeyError:
-            raise KeyError('Bond Dissociation energy not known for combination: '
-                           '({0},{1},{2})'.format(self.atom1.element.symbol, self.atom2.element.symbol, self.order))
+            if not self.is_surface_bond():
+                raise KeyError('Bond Dissociation energy not known for combination: '
+                               '({0},{1},{2})'.format(self.atom1.element.symbol, self.atom2.element.symbol, self.order))
+            else:
+                if self.atom1.is_surface_site():
+                    add_atom = self.atom2.element.symbol
+                else:
+                    add_atom = self.atom1.element.symbol
+                if self.is_van_der_waals():
+                    if add_atom == 'O':
+                        binding_energy = quantity.Energy(-0.189,'eV/molecule') # binding energy for H2O_ads in surfaceThermoPt111
+                    elif add_atom == 'N':
+                        binding_energy = quantity.Energy(-0.673,'eV/molecule') # binding energy for NH3_ads in surfaceThermoPt111
+                    elif add_atom == 'C':
+                        binding_energy = quantity.Energy(-0.122,'eV/molecule') # binding energy for CH4_ads in surfaceThermoPt111
+                    else:
+                        binding_energy = quantity.Energy(-0.2,'eV/molecule') # default vdw binding energy
+                    return -1 * binding_energy.value_si
+                try:
+                    from rmgpy.data.rmg import get_db
+                    tdb = get_db('thermo')
+                    metal_db = tdb.surface['metal']
+                except (DatabaseError, KeyError):
+                    from rmgpy.data.surface import MetalDatabase
+                    from rmgpy import settings
+                    metal_db = MetalDatabase()
+                    metal_db.load(os.path.join(settings['database.directory'], 'surface'))
+                metal_binding_energies = metal_db.find_binding_energies(metal=metal)
+                binding_energy = metal_binding_energies[add_atom]
+                max_bond_order = {'C': 4., 'O': 2., 'N': 3., 'H': 1.}
+                normalized_bond_order = self.order / max_bond_order[add_atom]
```

Comment:
That raises an interesting question. Linear Scaling Relations tell us that moving from one metal to another the _change_ in adsorption energy scales with the normalized bond order (eg. 1/2 for H2C=X and 1/4 for H3C-X) but is it true that the actual adsorption itself is 1/2 and 1/4 of the C binding energy respectively?  

I think the answer is "no", or else these lines would all have a _y_ intercept of 0 ?
![LSR slide](https://image.slidesharecdn.com/rmgcatlinearscaling-171031235043/95/incorporation-of-linear-scaling-relations-into-automatic-mechanism-generation-for-heterogeneous-catalysis-28-638.jpg?cb=1509544559)

Is the BDE we're after the adsorption energy, in this case?

(If it's just for use in the Blowers Masel fitting, I have found it's very insensitive PROVIDED that w_0 > 2 * E^0, and have used an arbitrary high value of 10000 kJ/mol. i.e. it's more important to be high enough than to be accurate; OTOH if we have a method called "get_bde" future users/developers may assume that's what it does, and we probably shouldn't return arbitrary numbers.).

Reply:
Ah yes, you're right. They would go through the origin. Looks like the 1/4 bond has small y intercept, but the 2/4 and 3/4 bon d are shifted down by about 1 to 1.5 eV. We could just use the equations of these lines to get a better estimate of BDE based on atomic binding energies. Maybe something like this for C-X bonds:
1/4 bond bde = 1/4 x
2/4 bond bde = 1/2 x - 1.35 eV
3/4 bond bde = 3/4 x - 1.08 eV
Where x is C binding energy in DB.